StudentAPI/Services/StudentService.cs

```diff
@@ -14,14 +14,20 @@ public interface IStudentService
     }
     public class StudentService : IStudentService
     {
+        private readonly DataContext _context;
+
+        public StudentService()
+        {
+            _context = new DataContext();
+        }
         public List<Student> GetAll()
         {
-            throw new NotImplementedException();
+            return _context.Student.ToList();
         }
 
         public Student GetById(int studentId)
         {
-            throw new NotImplementedException();
+            return _context.Student.Single(s => s.Id == studentId);
```

Comment:
Use `SingleOrDefault` instead of `Single`. Otherwise it will return an error if there are no records